TrashMob/client-app/src/components/ui/NavBar.tsx

```diff
@@ -79,12 +80,12 @@ const TopMenu: React.FC<TopMenuProps> = (props) => {
     return (
         <header className='tm-header'>
             <div className='container bg-light tm-mainNav'>
-                <div className='navbar navbar-expand-lg navbar-light navbar-static-top px-0' id='navbar'>
+                <div className='navbar  navbar-expand-lg navbar-light navbar-static-top px-0' id='navbar'>
                     <a className='navbar-brand' href='/' id='navbarBrand'>
-                        <img src={logo} alt='TrashMob Logo' className='logo m-0' />
+                        <img src={logo} alt='TrashMob Logo' className='logo m-0 h-100' />
```

Comment:
I don't think `h-100` is good here. 
I think we should update the  height `.logo` to match image aspect ratio, or make it auto 

Note that `.logo` is used in another pages too.

from 
```
.logo {
    width: 200px;
    height: 30px;
}
```

to 
```
.logo {
    width: 200px;
    height: auto; /** or height: 28px */
}
```

Reply:
This is another thing that I just quickfixed because the logo is set to be replaced with the new design. I just wanted something quick and dirty to fix the scaling on the navbar because it's ultimately going to be ripped out anyway.